test/cp.js

```diff
@@ -451,6 +451,45 @@ test('-R implies -P', t => {
   });
 });
 
+test('-Ru respects the -u flag recursively (don\'t update newer file)', t => {
+  const dir = `${t.context.tmp}/cp-Ru`;
+  const sourceDir = `${dir}/old`;
+  const sourceFile = `${sourceDir}/file`;
+  const destDir = `${dir}/new`;
+  const destFile = `${destDir}/file`;
+  [sourceDir, destDir].forEach(d => shell.mkdir('-p', d));
+  shell.ShellString('foo\n').to(sourceFile);
+  shell.ShellString('bar\n').to(destFile);
+  // Get the old mtime for dest
+  const oldTime = fs.statSync(destFile).mtimeMs;
+  // Send the old file to two days ago
```

Comment:
nit: you can better document this by creating a constant `TWO_DAYS_IN_MS`.

You could then make this comment more descriptive: `// Set the sourceFile to be older than destFile`.